### PR TITLE
perf(frontend): add non-sso template

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: templates/tab/js/non-sso/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@fluentui/react-northstar": "^0.54.0",
+        "@microsoft/mgt-element": "2.3.2-next.teamsfx.f5fc7d2",
+        "@microsoft/mgt-react": "2.3.2-next.teamsfx.f5fc7d2",
+        "@microsoft/mgt-teamsfx-provider": "2.3.2-next.teamsfx.f5fc7d2",
+        "@microsoft/microsoft-graph-client": "^3.0.1",
+        "@microsoft/teams-js": "^1.9.0",
+        "@microsoft/teamsfx": "^0.5.1",
```
Comment (SLdragon) [inline templates/tab/js/non-sso/package.json:12]: Use latest 0.6.0?

File: templates/tab/js/non-sso/src/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+  /* outline: 1px dashed #f00; */
```
Comment (SLdragon) [inline templates/tab/js/non-sso/src/index.css:3]: If it is not used, you can remove it

File: templates/tab/ts/non-sso/package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{
+    "name": "teamsfx-template-tab",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@fluentui/react-northstar": "^0.54.0",
+        "@microsoft/teams-js": "^1.9.0",
+        "@microsoft/teamsfx": "^0.5.1",
```
Comment (SLdragon) [inline templates/tab/ts/non-sso/package.json:8]: same here

File: templates/tab/js/non-sso/package.json (hunk 1 above)
Comment (KennethBWSong) [reply templates/tab/js/non-sso/package.json]: Updated. Thx!

File: templates/tab/ts/non-sso/package.json (hunk 3 above)
Comment (KennethBWSong) [reply templates/tab/ts/non-sso/package.json]: Updated. Thx!

File: templates/tab/js/non-sso/src/index.css (hunk 2 above)
Comment (KennethBWSong) [reply templates/tab/js/non-sso/src/index.css:3]: Here I prefer to keep the same as existing template.

Comment (SLdragon) [reply templates/tab/js/non-sso/src/index.css:3]: Sure

**Review by SLdragon — APPROVED** [review]

**Comment by kuojianlu** [comment]
Seems missing `.env` file.

Curious about: will this non-sso template replace the sso template as the default tab template? If so, when?

**Comment by KennethBWSong** [comment]
> Seems missing `.env` file.
> 
> Curious about: will this non-sso template replace the sso template as the default tab template? If so, when?

Seems `.env` file is ignored. Is it required? I tried local debug & provision, and seems it works fine.
The non-sso template will replace current template after we finish the AAD manifest feature before GA. Currently we have a feature flag for this feature.

File: templates/tab/js/non-sso/src/components/sample/EditCode.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+var functionName = process.env.REACT_APP_FUNC_NAME || "myFunc";
```
Comment (hund030) [inline templates/tab/js/non-sso/src/components/sample/EditCode.jsx:3]: Remove function related code?

**Review by hund030 — APPROVED** [review]

**Pull request merged**